bindings/python/gnucash_core.py

```diff
@@ -451,6 +455,144 @@ def __args_to_instance(args):
         else:
             raise TypeError('Required single int/float/str or two ints: ' + str(args))
 
+    # from https://docs.python.org/3/library/numbers.html#numbers.Integral
+    # and https://github.com/python/cpython/blob/3.7/Lib/fractions.py
+
+    def _operator_fallbacks(monomorphic_operator, fallback_operator):
+        def forward(a, b):
+            if isinstance(b, GncNumeric):
+                return monomorphic_operator(a, b)
+            if isinstance(b, int):
+                return monomorphic_operator(a, GncNumeric(b))
+            elif isinstance(b, float):
+                return monomorphic_operator(a, GncNumeric(b))
+            else:
+                return NotImplemented
+        forward.__name__ = '__' + fallback_operator.__name__ + '__'
+        forward.__doc__ = monomorphic_operator.__doc__
+
+        def reverse(b, a):
+            if isinstance(a, GncNumeric):
+                return monomorphic_operator(a, b)
+            elif isinstance(a, Rational):
```

Comment:
This reverse procedures remain a bit vague in comparison to forward() as here we look at Rational and Real.